### ISSUE TITLE
Question: Is there a way to browse my liked albums and artists?

### ISSUE THREAD

**Issue by s00ner** [marcdjulien/spotify-terminal#83]
Is there a way to browse through the albums and artists I've liked in Spotify? It looks like I can only really see my liked songs and playlists.

**Comment by marcdjulien** [comment]
Correct. No way to see Liked Artists or Albums. I can Implement soon in the next version.

For now, you'll have to find a track with the Artist or Album then select the "Goto Artist" (Shortcut: Shift + C) or "Goto Album" (Shortcut: Shift + X). Using "/" while in the Tracklist is a quick way to find a track.

**Comment by marcdjulien** [comment]
The [Spotify API](https://developer.spotify.com/documentation/web-api/reference/#category-playlists) only allows you to get a users Saved Playlists, Tracks, and Albums. So you won't be able to see Saved Artists.
